### ISSUE TITLE
Cargo.toml hasn't been updated for v0.3.3

### ISSUE THREAD

**Issue by dstaley** [portier/portier-broker#226]
I was testing out my v0.3.3 instance and was confused why it was reporting v0.3.2 even though I definitely downloaded the v0.3.3 release. Turns out the version reported on `/ver.txt` comes from `Cargo.toml`, which wasn't updated.

I was going to send in a PR, but I wasn't sure if just bumping the version in `Cargo.toml` was sufficient.

https://github.com/portier/portier-broker/blob/master/Cargo.toml#L3

**Comment by stephank** [comment]
Sorry, yeah, I botched that release a little bit. I'm not sure it's worth the trouble to redo it, but maybe we can think of something to prevent the mistake in the future.

Currently, the release process is:

1. Update the version in `Cargo.toml` and run `cargo check` to update `Cargo.lock`. Commit and push to `master` branch.
2. Create a release on GitHub from `master` branch. (Release notes are written manually.)
3. CI will automatically make release builds and attach them to the release.

I don't think there's a way to add validation to step 2, so maybe the only way to go is try to automate things more in scripts, so we then can have that validation?

**Comment by onli** [comment]
>  maybe the only way to go is try to automate things more in scripts, so we then can have that validation?

Automating as much as possible and otherwise having a clear recipe to follow is how we made releases possible again in another project I'm involved in were those were too hard before. So, this sounds good.

Maybe a bash script that does step 1 completely?


Something like:

```
./do_release.sh 0.x
```

```
#!/bin/bash
git checkout master   # not sure about this
sed "s/version = \".+\"/version = \"$1\"/" Cargo.toml
cargo check
git add .
git commit -v
git push
```

?

**Comment by stephank** [comment]
I like that idea! We can also automate the release on GitHub with the gh cli. I tried something in #228.

**Comment by stephank** [comment]
Think we can close this. Should be fixed since 0.3.4.
